.github/workflows/main.yaml

```diff
@@ -2,55 +2,16 @@ name: Devfile Docs Build
 
 on:
   push:
```

Comment:
IMO since we removed 

```
deploy_site:
    if: success() && github.ref == 'refs/heads/master'
```

from the workflow, we should only run this workflow on the `master` branch